include/z64save.h

```diff
@@ -12,44 +12,46 @@ struct FileSelectState;
+    /* 5 */ RESPAWN_MODE_ZORA,   // "RESTART_MODE_ZORA"
+    /* 6 */ RESPAWN_MODE_DEKU,   // "RESTART_MODE_NUTS"
+    /* 7 */ RESPAWN_MODE_HUMAN,  // "RESTART_MODE_CHILD"
     /* 8 */ RESPAWN_MODE_MAX
 } RespawnMode;
 
 #define SAVE_BUFFER_SIZE 0x4000
 
 typedef enum {
-    /* 0  */ MAGIC_STATE_IDLE, // Regular gameplay
+    /* 0  */ MAGIC_STATE_IDLE,          // Regular gameplay
     /* 1  */ MAGIC_STATE_CONSUME_SETUP, // Sets the speed at which the magic border flashes
-    /* 2  */ MAGIC_STATE_CONSUME, // Consume magic until target is reached or no more magic is available
+    /* 2  */ MAGIC_STATE_CONSUME,       // Consume magic until target is reached or no more magic is available
     /* 3  */ MAGIC_STATE_METER_FLASH_1, // Flashes border
     /* 4  */ MAGIC_STATE_METER_FLASH_2, // Flashes border and draws yellow magic to preview target consumption
-    /* 5  */ MAGIC_STATE_RESET, // Reset colors and return to idle
+    /* 5  */ MAGIC_STATE_RESET,         // Reset colors and return to idle
     /* 6  */ MAGIC_STATE_METER_FLASH_3, // Flashes border with no additional behaviour
-    /* 7  */ MAGIC_STATE_CONSUME_LENS, // Magic slowly consumed by Lens of Truth
+    /* 7  */ MAGIC_STATE_CONSUME_LENS,  // Magic slowly consumed by Lens of Truth
     /* 8  */ MAGIC_STATE_STEP_CAPACITY, // Step `magicCapacity` to full capacity
-    /* 9  */ MAGIC_STATE_FILL, // Add magic until magicFillTarget is reached
+    /* 9  */ MAGIC_STATE_FILL,          // Add magic until magicFillTarget is reached
     /* 10 */ MAGIC_STATE_CONSUME_GORON_ZORA_SETUP,
     /* 11 */ MAGIC_STATE_CONSUME_GORON_ZORA, // Magic slowly consumed by Goron spiked rolling or Zora electric barrier.
     /* 12 */ MAGIC_STATE_CONSUME_GIANTS_MASK // Magic slowly consumed by Giant's Mask
 } MagicState;
 
 typedef enum {
-    /* 0 */ MAGIC_CONSUME_NOW, // Consume magic immediately without preview
-    /* 1 */ MAGIC_CONSUME_WAIT_NO_PREVIEW, // Sets consume target but waits to consume. No yellow magic preview to target consumption. Unused
-    /* 2 */ MAGIC_CONSUME_NOW_ALT, // Identical behaviour to MAGIC_CONSUME_NOW. Unused
-    /* 3 */ MAGIC_CONSUME_LENS, // Lens of Truth consumption
-    /* 4 */ MAGIC_CONSUME_WAIT_PREVIEW, // Sets consume target but waits to consume. Show magic to be consumed in yellow.
-    /* 5 */ MAGIC_CONSUME_GORON_ZORA, // Goron spiked rolling or Zora electric barrier slow consumption
-    /* 6 */ MAGIC_CONSUME_GIANTS_MASK, // Giant's Mask slow consumption
-    /* 7 */ MAGIC_CONSUME_DEITY_BEAM // Fierce Deity Beam consumption, consumed magic now and not via request
+    /* 0 */ MAGIC_CONSUME_NOW,             // Consume magic immediately without preview
+    /* 1 */ MAGIC_CONSUME_WAIT_NO_PREVIEW, // Sets consume target but waits to consume. No yellow magic preview to
+                                           // target consumption. Unused
+    /* 2 */ MAGIC_CONSUME_NOW_ALT,         // Identical behaviour to MAGIC_CONSUME_NOW. Unused
```

Comment:
Uh oh, did you format z64save.h? We avoid formatting headers to avoid issues like this.

Reply:
Auto-format on save in the editor :-(, will restore original format.